src/electron/views/root-container/root-container-renderer.tsx

```diff
@@ -13,13 +13,10 @@ import {
 } from 'electron/views/root-container/components/root-container';
 import * as React from 'react';
 import * as ReactDOM from 'react-dom';
-import { ContentPanelDeps } from '../left-nav/content-panel-deps';
 
 export type RootContainerRendererDeps = RootContainerDeps &
     ThemeDeps &
-    PlatformBodyClassModifierDeps &
-    LeftNavDeps &
-    ContentPanelDeps;
```

Comment:
Removed LeftNavDeps and ContentPanelDeps from here because they are included in AutomatedChecksViewDeps, which is included in RootContainerDeps